utils/update-gatsby-fixtures.js

```diff
@@ -0,0 +1,111 @@
+const path = require('path');
```

Comment:
Can we do TypeScript for these files and invoke with `ts-node`?

Reply:
The github action thing we use to execute doesn't have typescript support it seems https://github.com/actions/github-script/issues/294

Is this a blocker? I we use this pattern in vercel/api as well as the other cron scripts here in this repo so figuring out typescript support seems a bit out of scope